src/main/java/cookie/thaumaturgy/api/Dunami.java

```diff
@@ -7,32 +7,32 @@ public class Dunami {
 	// Unofficial plural (not actual greek) of Dunamis.
 	public static final List<Dunamis> DUNAMI = new ArrayList<>();
 
-	public static final Dunamis AIR = new DunamisBuilder("air")
+	public static final Dunamis AIR = new DunamisBuilder("Air")
```

Comment:
Internal dunamis names should be camelCase.
Use translations for displaying the name in-game.